controllers/past-grants/index.js

```diff
@@ -117,46 +117,15 @@ router
         const queryWithPage = addPaginationParameters(facetParams, req.query.page);
         const data = await queryGrantsApi(queryWithPage);
 
-        const commonSortOptions = [
-            {
-                label: 'Oldest first',
-                value: 'awardDate|desc'
-            },
-            {
-                label: 'Lowest amount first',
-                value: 'amountAwarded|asc'
-            },
-            {
-                label: 'Highest amount first',
-                value: 'amountAwarded|desc'
-            }
-        ];
+        let currentSort = { type: 'awardDate', direction: 'desc' };
```

Comment:
I feel like this should come from the API rather than here? Or could we send this parameter as an actual `?sort` option, so we know it's definitely the default (in case we change this at the API end)?